### PR TITLE
Moved Documention content to Community resources

### PR DESCRIPTION
**Pull request by SafeEHA**
Needs review
cc: @elfosardo

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
         <ul id="main_nav" class="mk-main-nav">
           <li {% if page.url == "/blog/index.html" %} class="active" {% endif %}><a class="mk-main-nav__item" href="{{ site.baseurl }}/blog/index.html">Blog</a></li>
           <li {% if page.url == "/community-resources.html" %} class="active" {% endif %}><a class="mk-main-nav__item" href="{{ site.baseurl }}/community-resources.html">Community Resources</a></li>
-          <li {% if page.url == "/documentation.html" %} class="active" {% endif %}><a class="mk-main-nav__item" href="{{ site.baseurl }}/documentation.html">Documentation</a></li>
+          <li {% if page.url == "/documentation.html" %} class="active" {% endif %}><a class="mk-main-nav__item" href="https://book.metal3.io/introduction.html">Documentation</a></li>
```
Comment (elfosardo) [inline _includes/header.html:32]: href should be just https://book.metal3.io

File: _includes/footer.html [hunk 2]
```diff
@@ -47,7 +47,16 @@
 </script>
     <script src="/assets/js/copy.js"></script>
     <!-- This comes from DTM/DPAL and must be latest entry in body-->
+<script>
+  let pageLocation = window.location.href
+  let footerIcons = document.querySelector(".mk-icons-footer")
 
+  if(pageLocation.includes("community-resources.html")){
+    console.log ("Yes")
```
Comment (mboukhalfa) [inline _includes/footer.html:55]: ?

File: assets/images/youtube.svg
Comment (mboukhalfa) [inline assets/images/youtube.svg:1]: where this file is used ?

File: _includes/footer.html (hunk 2 above)
Comment (SafeEHA) [reply _includes/footer.html:55]: Please can you clarify your question?

File: assets/images/youtube.svg
Comment (SafeEHA) [reply assets/images/youtube.svg:1]: the file is used on the community resources page

File: _includes/footer.html (hunk 2 above)
Comment (mboukhalfa) [reply _includes/footer.html:55]: Please remove if it was for debugging

File: assets/images/youtube.svg
Comment (mboukhalfa) [reply assets/images/youtube.svg:1]: Can you please link to where it was used ? I could not find it mentioned anywhere in community resources the icon is an embedded svg content in the html

Comment (SafeEHA) [reply assets/images/youtube.svg:1]: It is the last link in the "Get Connected Section" https://metal3.io/community-resources.html

Comment (mboukhalfa) [reply assets/images/youtube.svg:1]: Is not there the following content create the icon without need to the file ?
```
<svg style="height:15px; width:15px" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 461.001 461.001" xml:space="preserve">
  <g>
    <path style="fill:var(--mk--color-brand--400);" d="M365.257,67.393H95.744C42.866,67.393,0,110.259,0,163.137v134.728
                       c0,52.878,42.866,95.744,95.744,95.744h269.513c52.878,0,95.744-42.866,95.744-95.744V163.137
                       C461.001,110.259,418.135,67.393,365.257,67.393z M300.506,237.056l-126.06,60.123c-3.359,1.602-7.239-0.847-7.239-4.568V168.607
                       c0-3.774,3.982-6.22,7.348-4.514l126.06,63.881C304.363,229.873,304.298,235.248,300.506,237.056z" />
  </g>
</svg>
```

Comment (SafeEHA) [reply assets/images/youtube.svg:1]: Oh! Alright, I understand 
Would take out the file now

Comment (SafeEHA) [reply assets/images/youtube.svg:1]: It is ok now i believe, anything else i need to change?

File: _includes/header.html (hunk 1 above)
Comment (elfosardo) [reply _includes/header.html:32]: this still needs to be changed

**Pull request merged**